sunpy/net/tests/test_fido.py

```diff
@@ -183,18 +184,18 @@ def test_fetch():
 def test_unifiedresponse_slicing():
     results = Fido.search(
         a.Time("2012/1/1", "2012/1/5"), a.Instrument.lyra)
-    assert isinstance(results[0:2], UnifiedResponse)
-    assert isinstance(results[0], UnifiedResponse)
+    assert isinstance(results[0:2], BaseQueryResponse)
+    assert isinstance(results[0], BaseQueryResponse)
```

Comment:
Is `results` a `UnifiedResponse`?

Reply:
yes, the `__getitem__` returns a `BaseQueryResponse` instance if there is record from only one client in a fido result, as we decided in the meeting.

Reply:
Can you just assert that in the test as well?

Reply:
done.

Reply:
Thanks.